bin/mozilla-en-US.py

```diff
@@ -117,45 +112,33 @@ def push(path):
 dname = os.path.dirname(abspath)
 os.chdir(dname)
 
-for channel in ['beta', 'central']:
-    for repo in SOURCE_REPOS:
-        ending = repo + '-' + channel
-
-        url_folder = ''
-        if channel == 'central':
-            if repo == 'mozilla':
-                url_folder = 'l10n/'
-        else:
-            url_folder = 'releases/'
-
-        url = 'ssh://hg.mozilla.org/' + url_folder + ending
-        target = os.path.join('source', ending)
-
-        # Clone or update source repositories
-        pull(url, target)
-
-    for repo in TARGET_REPOS.keys():
-        ending = repo + '-' + channel
-        url = 'ssh://hg.mozilla.org/users/m_owca.info/' + ending
-        target = os.path.join('target', ending)
+# Clone or update source repository
+url = 'ssh://hg.mozilla.org/users/axel_mozilla.com/gecko-strings/'
+target = 'source'
+pull(url, target)
 
-        # Clone or update target repositories
-        pull(url, target)
+for repo in TARGET_REPOS.keys():
+    ending = repo + '-central'
+    url = 'ssh://hg.mozilla.org/users/m_owca.info/' + ending
+    target = os.path.join('target', ending)
 
-        # Copy folders from source to target
-        folders = TARGET_REPOS[repo]['folders']
-        source = TARGET_REPOS[repo]['source'] + '-' + channel
+    # Clone or update target repository
+    pull(url, target)
 
-        for folder in folders:
-            origin = os.path.join('source', source, folder, 'locales/en-US')
-            destination = os.path.join('target', ending, folder)
+    # Prune all subdirectories in target repository in case they get removed from source
+    for root, subdirs, files in os.walk(target, True):
```

Comment:
Using os.walk is a bit excessive? How about just listdir? 'cause I think that's what you're ending up with?